src/System.Xml.ReaderWriter/tests/XmlWriter/System.Xml.RW.XmlWriter.Tests.csproj

```diff
@@ -29,6 +29,10 @@
       <Project>{be8ed8c1-c314-4c4e-a929-64c9c8b3552a}</Project>
       <Name>XunitTraitsDiscoverers</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\System.Text.Encoding.CodePages\src\System.Text.Encoding.CodePages.csproj">
```

Comment:
They will remember if there is still the active issue assigned to them. In general I know it is very tempting to want to reference the live bits for everything but we have to be careful as we want to make sure the dependencies are explicitly defined via the packages. We are trying to move to the model were each library package can ship independently and as such we have to be only depend upon what the package says it depends on and not accidentally take dependencies on newer versions. 

It is unfortunate that we are in this current state however were we don't have a good on demand way of updating dependent packages, but we will get there at some point.